src/Orleans.Reminders/Providers/ReminderProviderRuntime.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Runtime;
+using Microsoft.Extensions.Logging;
+using Orleans.Reminders;
+using System.Threading;
+using Orleans.Runtime.ReminderService;
+using Orleans.Hosting;
+using Microsoft.Extensions.Options;
+
+namespace Orleans.Providers
+{
+    internal class ReminderProviderRuntime : IReminderProviderRuntime, ILifecycleParticipant<IClusterClientLifecycle>
```

Comment:
I wonder if this is necessary. Maybe it can be removed altogether

Reply:
removed this